recipes-kernel/linux/linux-mainline_5.2.bb

```diff
@@ -1,34 +1,27 @@
 require recipes-kernel/linux/linux-mainline-common.inc
 
-LINUX_VERSION ?= "5.1.x"
+LINUX_VERSION ?= "5.2.x"
 KERNEL_VERSION_SANITY_SKIP="1"
 
-BRANCH = "linux-5.1.y"
+BRANCH = "master"
 SRCREV = "${AUTOREV}"
 SRC_URI = " \
     git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git;branch=${BRANCH} \
 "
 
 # Out of tree patches required for HiFive Unleashed
 SRC_URI_append_freedom-u540 = " \
-    file://0001-tty-serial-add-driver-for-the-SiFive-UART.patch \
```

Comment:
If this is removed don't we need the upstream drivers? That means we need the upstream device tree, how is that being passed to the kernel?

Reply:
> ****
> Use the 5.2-rcX series kernels with new (stable) device tree format(s)

Won't this break u-boot as it has the new bindings? How did you test this?